docs/app/pages/Components/List/List.vue

```diff
@@ -39,6 +39,11 @@
         <li v-for="event in interactionEvents" :key="event">{{ event }}</li>
       </ul>
 
+      <note-block>
+        Mixing a list action with actions on the list items hurts usability. When clicking on the list item action, will the list action be triggered as well?
```

Comment:
I found this tip a little confusing. Maybe we can have more examples to show why this will happen. What do you think?

Reply:
I am not sure, maybe better will be to link to issue?

Reply:
Related to #1653.

Probable solution #1680.

Reply:
@VdustR yes i know but i mean linking in docs to that issues will be better then examples

Reply:
I agree about `<div>` if there is no better way to implement that.